addons/account/models/account_move.py

```diff
@@ -2264,6 +2264,7 @@ def is_internal_partner(partner):
 
         # Create the invoice.
         values = {
+            'name': self.default_get(['name'])['name'],
```

Comment:
Why not simply '/'?

```suggestion
            'name': '/',
```

Reply:
Yes, I also added a `_compute_name()` and added a small test

Reply:
`_compute_name()` is needed for the first invoice of a tuple (journal, period)

Reply:
isn't it the post() method that does the computation, even for the first invoice per tuple (journal, period) ? The invoice/bill will still be 'draft' after its creation, so I believe your test is wrong as well :thinking:

Reply:
No, the first invoice of the tuple (journal, period) already has a name in draft so that you can edit it in the form view

Reply:
pffeeww... we're doing too smart things those days!

would you mind adding a comment explaining that for the next lambda guys asking himself why the compute_name isn't done by the post ?

wait a minute... it should be done by the create() otherwise we could/should have the same bunch of code everywhere we create a new entry  :thinking:

Reply:
It should be done by the create but here we have to give a value for `name` otherwise the `message_new` gives the mail subject by default.
Because we give a value, it is not recomputed (that's how stored computed fields work). Hence, I have to recompute it by hand.
Usually, nothing has to be done because the compute is triggered by the `post` method or by the create method (because we don't **have to** give the name, it is even bad practice)

What we could do is remove the name from the create dict (inside the `create` method)?